components/bms_worker/src/Environment.c

```diff
@@ -251,8 +251,8 @@ void ENV_calcTempStats(void)
     uint8_t connected_channels = 0;
 
     ENV.values.avg_temp = 0;
-    ENV.values.max_temp = INT16_MIN;
```

Comment:
i set values of -200, as 'impossible' to reach instead of INT16_MIN since max_temp is now a float

Reply:
rather than setting some arbitrary float as invalid, you should consider adding a `valid` bit to this struct which you can check separately from the contents of these floats.